components/FileUploadField.tsx

```diff
@@ -0,0 +1,50 @@
+import { AttachmentIcon } from '@chakra-ui/icons';
+import { Button } from '@chakra-ui/react';
+import { SyntheticEvent, useRef } from 'react';
+
+// File upload button props
+type FileUploadProps = {
+  currentFile: File | null; // currently uploaded file
+  onUploadFile: (selectedFile: File) => void; // handle file upload
+};
+
+//TODO: rename this better??
```

Comment:
Let's call this `PoaFormUploadField` and put it in `components/admin/requests/guardian-information/PoaFormUploadField.tsx`